mflowgen/Tile_MemCore/construct.py

```diff
@@ -366,6 +366,10 @@ def construct():
       postroute.extend_postconditions(    ["assert 'Clamping logic structure in the SBs and CBs is maintained' in File( 'mflowgen-run.log' )"] )
       signoff.extend_postconditions(      ["assert 'Clamping logic structure in the SBs and CBs is maintained' in File( 'mflowgen-run.log' )"] )
    
+  # The presence of SDF registers after synthesis means adk got corrupted somehow.
+  # The registers cause short circuits that are not found until much later, after LVS.
+  # This has happened several times now, see .e.g. https://buildkite.com/tapeout-aha/fullchip/builds/358
+  synth.extend_postconditions( ["assert 'SDFQ' not in File( 'outputs/design.v' )"] )
```

Comment:
I agree that the test is technology-specific, but then of course `construct.py` is technology-specific, see above where it says `adk_name = 'tsmc16'` :)

But I do see your point. I could change the test to something like
```
if (adk_name == "tsmc16"); then extend_postconditions ...
```
and/or if you guarantee that the problem will never happen again, I can take out the test(s) entirely...otherwise I see no harm in keeping them...

Reply:
There's one line that selects the ADK, but ideally we keep the rest of the code technology agnostic here. I'd rather remove it entirely.

The other issue here is that this check is only being added to the mem and PE flows, but of course, it's not mem or PE specific. Ideally, if we wanted to add this check, we'd add it across all of our flows. But the way mflowgen is right now makes that tedious. Some changes are coming that should hopefully help with that. Regardless, I don't think we need this check anyway.

Reply:
Removed.